addon/utils/decorator-wrapper.js

```diff
@@ -0,0 +1,26 @@
+import isDescriptor from './is-descriptor';
+import normalizeDescriptor from './normalize-descriptor';
+
+function handleDescriptor(target, key, desc, fn, params = []) {
+  normalizeDescriptor(desc);
+
```

Comment:
These are copied over from `ember-decorators`.

Reply:
@buschtoens actually just finished pulling out these functions into the `@ember-decorators/utils` library, and I published the changes this morning as `v0.2.0`, so I think it would be better to use those and avoid duplication